src/helpers/utils.ts

```diff
@@ -94,6 +94,7 @@ export const getRequestBuilder = (options: RequestOptions) => {
 
     if (options.headers !== undefined) {
       options.headers.forEach((value, key) => {
+        // @ts-ignore
```

Comment:
why do we need this?

Reply:
Typescript was complaining about an `any` type here and this seemed like the simplest solution as the code was taken from datadog-ci and I wasn't very familiar with it.

Reply:
It's probably better to add an explicit type. If you're not sure what that is, you can use an any type.

Reply:
Done!